libs/core/algorithms/include/hpx/parallel/algorithms/adjacent_find.hpp

```diff
@@ -15,8 +15,8 @@
 namespace hpx {
     /// Searches the range [first, last) for two consecutive identical elements.
```

Comment:
line 48 should be:

`template <typename InIter, typename Pred = hpx::parallel::v1::detail::equal_to>`